louisewoodhams/LOUISE_WOODHAMS.md

```diff
@@ -0,0 +1,78 @@
+if can't it should supply a 400 Bad or something in the 400 range. They should never be created and used on purpose. 500 is just not ok which is why they make me so happy when I find them.
+
+
+## Postman 🚀
+
+I mentioned Postman briefly earlier on. Honestly, when I first started testing APIs, I installed it (https://www.postman.com/downloads/) set it to darkmode and went back to the Swagger as I found it too overwhelming. 
+If the API you are testing is simple or you can't get on with Swagger, Postman is the way to go. 
+I love using Postman now and not only for how easy it makes everything but it is a great place to store little API calls you only use occasionally and a fantastic playground to set up a API test collection that you could eventually add into your pipeline. It has amazing scope for writing tests with a wealth of test code snippets and environment options to help you along the way. 
+It handles everything Swagger does in an easier to use way with more future potential but first time it can be too much in one go. 
+ 
+
+## Mocking And Shouting BEES Beeceptor🐝
+
+During my time API testing, other than 500s the thing that has bought me the most joy is Beeceptor https://beeceptor.com/. Similar to Mocky, Beeceptor allows you to stand up a URL Endpoint to intercept your requests and send on any custom response you want.  It doesn't require much setup and is a great way to mock a client request or help check a process beyond your API will work. I had to use this frequently for the first API I tested. It had a lot of moving parts and required notification functions to be run for to local systems. It often took me a while to set everything up and trial and error but when the notifications came through to Beeceptor and it sent on its message I couldn't help but shout "BEES" in glee. After months of this me and Beeceptor are firm friends. It even helped me with a rather unpleasant SOAP API process I had to debug.
+
+## When It Gets SOAPY And Final API Thoughts
+
+I feel unlucky as I have had to deal with SOAP a few times in my career. It is REST APIs shameful older brother and the less said about it the better. It has its own rules for error handling and isn't really a fan of being helpful in anyway in my experience. SOAP requests are in XML which is not much fun to write either and must easier to get wrong.
+
+I hope some of the things I have said have been useful or at least given you a solid place to start with your REST API testing journey.  You now know a little more at least and most of all feel more prepared to tackle an API if a wild one appears and casts confusion, it won't be super effective. I am by no means an expert but all experience, good or bad helps shape us and any knowledge we can pass on can be useful one way or another. 
+
+
+
```

Comment:
Do you want to add a Bio section?

Reply:
Not sure, the only story I reviewed didn't have one.